src/client.js

```diff
@@ -721,6 +721,15 @@ client.prototype.handleMessage = function handleMessage(message) {
               //
               break;
 
+            case 'msg_rejected':
+              this.log.info(`[${channel}] ${msg}`);
+              this.emit('automodinreview', channel);
```

Comment:
Can we emit `{ channel }` here so that if we needed to emit additional properties, we won't have to worry about argument order?

It is planned to refactor all events so that they emit a single object like this.